mkt/search/views.py

```diff
@@ -101,6 +102,20 @@ class NonPublicSearchView(SearchView):
                        ProfileFilter, SortingFilter]
 
 
+class NoRegionSearchView(SearchView):
+    """
+    A search view that allows searching for public apps regardless of region
+    exclusions, protected behind a permission class.
+
+    """
+    authentication_classes = [RestSharedSecretAuthentication,
+                              RestOAuthAuthentication]
+    permission_classes = [GroupPermission('Feed', 'Curate')]
+    filter_backends = [SearchQueryFilter, PublicSearchFormFilter,
+                       PublicAppsFilter, DeviceTypeFilter,
```

Comment:
The client is not sending the device or profile parameter though, so do we care ?